remote/remote.go

```diff
@@ -32,6 +35,9 @@ var log = golog.Logger("remote")
 // hook contexts may be populated with request parameters
 type Hook func(ctx context.Context, pid profile.ID, ref dsref.Ref) error
 
+// OptionsFunc is a type of function that takes an Options
+type OptionsFunc func(o *Options)
```

Comment:
```suggestion
// RemoteOptionsFunc adjusts the behavior of Remote when passed to NewRemote
type RemoteOptionsFunc func(o *Options)
```

No more fight left in me on the spelling of `behavior`. 😄